tests/integrationv2/test_well_known_endpoints.py

```diff
@@ -81,14 +81,13 @@ def test_well_known_endpoints(managed_process, protocol, endpoint, provider, cip
     else:
         client_options.trust_store = "../integration/trust-store/ca-bundle.crt"
 
-    client = managed_process(provider, client_options, timeout=5)
+    client = managed_process(provider, client_options, timeout=5, expect_stderr=True)
```

Comment:
Why is `expect_stderr=True` here -  What error is s2n throwing?

Reply:
https://github.com/aws/s2n-tls/blob/main/bin/echo.c#L180-L184 :D 
Might be worth changing, but I'd argue out of the scope of this PR.

Reply:
Ahh..could you add a comment and track it  in an issue to make sure we fix it? I dont want us forgetting the reason why we have it as true here.

Reply:
I don't see any evidence it needs to be fixed. It could be expected behavior.